src/DialogWindow.vala

```diff
@@ -131,10 +144,18 @@ namespace Ilia {
                         return true;
                     }
                     if (key.keyval == 'j' || key.keyval == 'n') { // Down - vim/emacs style
+                        if (is_on_help_page && help_scrolled_window != null) {
+                            scroll_help_window(true);
+                            return true;
+                        }
```

Comment:
This can be handled cleanly with a seperate module or widget for the `HelpPage`.